test/integration/verify_up4.go

```diff
@@ -19,6 +19,7 @@ const (
 	// TODO: auto-generate P4 constants from P4Info and share them with p4rt_translator
 	MeterSession              = "PreQosPipe.session_meter"
 	MeterApp                  = "PreQosPipe.app_meter"
+	MeterSliceTc              = "PreQosPipe.slice_tc_meter"
```

Comment:
I suggest starting to use constants from `internal/p4constants` in the tests too. We should refactor tests, but at least let's not to introduce any new variable here.